### DISCUSSION TITLE
11-digit recognition laughably bad for LCD displays?

### DISCUSSION THREAD

**Discussion by joenye** [jomjol/AI-on-the-edge-device#2042] [General]
### The Problem

I am getting a lot of errors with what looks to me like very clear examples. See screenshot below for example.

Is this due to lack of training data with LCDs? Seems to have very poor accuracy.

### Version

Release: v14.0.3 (Commit: 6c891ab+)

### Logfile

```shell
N/A
```

### Screenshots

![image](https://user-images.githubusercontent.com/2231018/219108115-278c2890-6844-4699-b0c6-31e03c2f44ed.png)

**Comment by friedpa** [comment]
I think it is about the very bad contrast, maybe you can adjust the brightness and also  the sharpness of the picture. Try to get rid of the reflection in the middle with a black sticker.

**Comment by Fusseldieb** [comment]
Never used this firmware before, but can you turn up the contrast of the camera? It should help.

**Comment by joenye** [comment]
Thanks for the advice. I have improved the contrast and this seemed to help a little.

The other thing which seems to have made perhaps the most difference is ensuring the inner ROI box fully encompasses each digit. Each digit looks like this now and so far so good:

![image](https://user-images.githubusercontent.com/2231018/219216901-34e40c9f-6ba9-4aba-a242-83c6ba658784.png)
